src/esargent.java

```diff
@@ -0,0 +1,129 @@
+import java.util.Scanner;
+
+class esargent {
+	
+	private double calcDouble(String[] plusArray) {
```

Comment:
When you look at calcDouble and calcLong, you can see a large amount of redundancy. These are perfect candidates for review and revision since they share so much common code. If you were to combine the two you could likely have the same effect with much less code.